IDE/Renesas/e2studio/GR-ROSE/test/src/test_main.c

```diff
@@ -0,0 +1,256 @@
+/* test_main.c
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+//#include "typedefine.h"
+#ifdef __cplusplus
+//#include <ios>                        // Remove the comment when you use ios
```

Comment:
This comment mentions ios.  Will TSIP users be using iOS?

Reply:
I agree TSIP will not be used on iOS and those commented lines should be removed.